container/overlay/docker-compose.yaml

```diff
@@ -5,18 +5,112 @@ version: "2.4"
 
 services:
 
-  aio-demo:
-    image: aio-demo
-    container_name: aio-demo
-    expose:
-      - "8080"
-    ports:
-      - "8080:8080"
-    build:
-      context: .
-      dockerfile: Dockerfile
-      args:
-        - _JAVA_OPTIONS
-    environment:
-      - "_JAVA_OPTIONS=${_JAVA_OPTIONS}"
-    network_mode: host
+    redis:
```

Comment:
If you want run the AIO demo, then please cherry-pick [ Update AIO to use Key-blob | 7c1cd5e ].